pkg/credentials/gcp.go

```diff
@@ -0,0 +1,32 @@
+package credentials
+
+import "encoding/json"
+
+type gcpCredAccountKeys string
+
+// From https://github.com/golang/oauth2/blob/d3ed0bb246c8d3c75b63937d9a5eecff9c74d7fe/google/google.go#L95
```

Comment:
here could wee use it directly? i think we already use it as indirect dependecy https://github.com/openshift/oadp-operator/blob/master/go.mod#L136

Reply:
If a var or const is small case, it is not exported, not accessible by external package and modules like ours.

Reply:
Oh yeah, forgot about that :man_facepalming: